mrblib/mrb_mrbgem_template.rb

```diff
@@ -19,7 +19,8 @@ def initialize(params = {})
     #  :class_name     => 'Hogehoge',
     #  :author         => 'mruby-hogehoge developers',
     #  :ci             => false, # default to :default, :matrix also available
-    #  :bin_name       => 'foocli' | true # if true,detect bin name by gem name
+    #  :bin_name       => 'foocli' | true, # if true,detect bin name by gem name
+    #  :mruby_version  => '3.1.0'
```

Comment:
The option `mruby_version` existed but was undocumented.

https://github.com/matsumotory/mruby-mrbgem-template/blob/4159eb217a3d6bd0ade898fee57e78a8f92805b3/mrblib/mrb_mrbgem_template.rb#L33-L35

Reply:
Sorry, I missed `options.rb`. `mruby_version` is documented since before...

https://github.com/matsumotory/mruby-mrbgem-template/blob/4159eb217a3d6bd0ade898fee57e78a8f92805b3/mrblib/options.rb#L46-L48